tests/integration/jsonrpc_commands_goerli.json

```diff
@@ -1312,8 +1312,8 @@
             "id":1,
             "jsonrpc":"2.0",
             "error":{
-                "code":-32602,
-                "message":"invalid eth_sendRawTransaction encoded tx: 0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f07244567"
+                "code":-32000,
+                "message":"rlp: unknown tx type prefix"
```

Comment:
This change in the error message is due to some change in Silkworm RLP encoding error handling, isn't it?